webpack/assets/javascripts/foreman_ajax_remount.js

```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';
+
+/* eslint-disable prefer-arrow-callback, func-names */
+
+$(document).on('RemountComponents', function() {
+  window.tfm.ajaxRemount.remountComponents();
+});
+
+export function remountComponents() {
+  const range = document.createRange();
+  range.selectNode(document.getElementsByTagName('body').item(0));
+
+  $('.react_mounted script').each(function() {
```

Comment:
I couldn't find any instance of `.react_mounted` where do we use it?

Reply:
I believe, that @xprazak2 preparing it for his plugin.